src/openms/source/METADATA/ProteinIdentification.cpp

```diff
@@ -81,7 +81,8 @@ namespace OpenMS
     enzyme(UNKNOWN_ENZYME),
     missed_cleavages(0),
     peak_mass_tolerance(0.0),
-    precursor_tolerance(0.0)
+    precursor_tolerance(0.0),
+    digestion_enzyme("unknown_enzyme","")
```

Comment:
it's making sense to store it as a pointer. I didn't know which one is better and struggled a little bit. Thanks for bringing it up.